index.js

```diff
@@ -55,20 +57,28 @@ function importFromJsonString(idbDatabase, jsonString, cb) {
   const importObject = JSON.parse(jsonString);
   Array.from(idbDatabase.objectStoreNames).forEach((storeName) => {
     let count = 0;
-    Array.from(importObject[storeName]).forEach((toAdd) => {
-      const request = transaction.objectStore(storeName).add(toAdd);
-      request.onsuccess = () => {
-        count++;
-        if (count === importObject[storeName].length) {
-          // added all objects for this store
-          delete importObject[storeName];
-          if (Object.keys(importObject).length === 0) {
-            // added all object stores
-            cb(null);
+    if (importObject[storeName] && Array.from(importObject[storeName]).length > 0) {
+      Array.from(importObject[storeName]).forEach((toAdd) => {
```

Comment:
I think there should be a variable introduced to avoid repeating `Array.from(importObject[storeName])`